src/App.js

```diff
@@ -1,26 +1,70 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import Home from './home';
+import Cart from './cart';
+import {
+  Navbar,
+  Nav,
+  NavItem,
+} from 'reactstrap';
+import 'bootstrap/dist/css/bootstrap.css';
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
+import { productData } from "./data/index"
+export default class App extends Component {
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+  constructor(props) {
+    super(props)
+    this.state = ({
+      data: null,
+      cart: []
+    })
+  }
+
+  getDataAfterAdd = (data) =>{
+    const {cart} = this.state;
+    console.log(cart)
```

Comment:
Please remove this `console.log` as it's not important and sometime it can be a serious thing since we are logging our item out to client.